tools/diff-processor/breaking_changes/resource_diff_test.go

```diff
@@ -88,3 +107,112 @@ var resourceSchemaRule_RemovingAField_TestCases = []resourceSchemaTestCase{
 		expectedFields: []string{"field-a", "field-b"},
 	},
 }
+
+var resourceSchemaRule_AddingExactlyOneOf_TestCases = []resourceSchemaTestCase{
+	{
+		name: "control",
```

Comment:
nit: control is a slightly uncommon term (and we might want to add more base cases later)

```suggestion
		name: "no changes",
```